.eslintrc.cjs

```diff
@@ -18,6 +18,7 @@ module.exports = {
 		},
 		{
 			extends: [
+				"plugin:jsdoc/recommended-typescript",
```

Comment:
> Adding emphasis:
> 
> > chore: Use eslint-plugin-jsdoc's **recommended-typescript-error** ruleset
> 
> Close but not quite the right ruleset 😄. Search for `typescript` in [gajus/eslint-plugin-jsdoc#configuration](https://github.com/gajus/eslint-plugin-jsdoc#configuration).

This is not the ruleset I asked for. 😞 

Do you see the difference?

Reply:
sigh, yes. this has been updated.